sdk/eventhub/event-hubs/src/eventHubClient.ts

```diff
@@ -394,7 +425,7 @@ export class EventHubClient {
 
   /**
    * @property
-   * The name of the default consumer group for any Event Hub instance
+   * The name of the default consumer group in the Event Hubs service.
```

Comment:
We should update the name of this static member to `defaultConsumerGroupName`

In the future, when we may allow users to change the name of the default consumer group, having something to reflect that this is the default **name** can help. Check what other languages are doing

Reply:
Hadn't considered it, but this makes sense to me.   I just made the same change in the .NET library.

Reply:
Changed!